nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -1041,12 +1052,46 @@ def export_model(self, model_path, calibration_path=None, onnx_path=None, input_
 
         return calibration_config
 
+    def fold_bn(self, *inputs, wrapper):
```

Comment:
What about moving this function to the superclass, so that QAT_quantizer and LsqQuantizer can both use it

Reply:
Done.